### ISSUE TITLE
USB to 1000BASE-SX / USB to 10GBASE-SR

### ISSUE THREAD

**Issue by robert-scheck** [fabiobaltieri/usb-100basefx#1]
Do you have plans for something like "USB to 1000BASE-SX" or "USB to 10GBASE-SR"? Depending on the exact use case this requires USB 3.1 Gen 2.

**Comment by fabiobaltieri** [comment]
Not really. I wanted to do a gigabit version but the only option I've found was a combination of an RGMII Asix chip (the AX88178 I think) plus a Vitesse VSC8211 PHY that goes into SerDes for an SFP module. It should work but the PHY only comes in fairly unfriendly 117 pin rectangular BGA package. Plus the MAC is USB 2.0 only. Too messy...
